.github/workflows/codeql-analysis.yml

```diff
@@ -14,7 +14,8 @@ on:
 
 jobs:
   codeql-analysis:
-    uses: wultra/wultra-infrastructure/.github/workflows/codeql-analysis.yml@develop
+    uses: wultra/wultra-infrastructure/.github/workflows/codeql-analysis.yml@issues/31-internal-repo-codeql
```

Comment:
This link is for testing purposes only. This line should be reverted back before merging